foundation/aissemble-foundation-core-python/tests/features/steps/policy_test_util.py

```diff
@@ -106,10 +106,15 @@ def getRandomRules(number: int) -> List[PolicyRuleInput]:
         return rules
 
     @staticmethod
-    def getRandomTarget() -> Target:
-        retrieveUrl = "http://" + PolicyTestUtil.getRandomString(5) + ".com"
-        type = PolicyTestUtil.getRandomString(5)
-        return Target(retrieve_url=retrieveUrl, type=type)
+    def getRandomTargets(number: int) -> List[Target]:
+        targets = []
+
+        for index in range(number):
+            retrieveUrl = "http://" + PolicyTestUtil.getRandomString(5) + ".com"
+            type = PolicyTestUtil.getRandomString(5)
+            targets.append(Target(retrieve_url=retrieveUrl, type=type))
+
+        return targets
```

Comment:
Q: (this isn't really about your PR) but not fully following why we originally had the random component / what this is accomplishing?

Reply:
I guess the idea was to make each target unique. Doesn't really help or harm the testing as is.